build.gradle.kts

```diff
@@ -1,19 +1,3 @@
-buildscript {
-    repositories {
-        mavenCentral()
-        google()
-        gradlePluginPortal()
-    }
-    dependencies {
-        classpath(libs.com.android.tools.build.gradle)
```

Comment:
We don't need this, because it's already in plugin classpath, leaking from buildSrc.
[Here](https://github.com/KakaoCup/Kakao/pull/109/files#diff-7b5c40eb5522b5216cfe8c75612e579a8db0daee9ded80df9ce6a56e8361a962R7)